### ISSUE TITLE
Duplex printing with a Brother MFCL3770CDW

### ISSUE THREAD

**Issue by lafontaj** [OpenPrinting/cups-filters#506]
**Describe the bug**
Hi, 

I was referred here to submit a bug report

English is not my first language, please do not do joke, I will probably not understand.  I'm not very used to fill bug report.

I just notice a problem with my Brother MFCL3770CDW printer (Ethernet Network) with Mint 21.1 OS

I did not install any drivers, Mint 21.1 discovered the network printer and the test page was OK.

In LibreOffice or XReader, it's possible to request a print in duplex mode all choice are possible as it should be. I can launch the print, but the result is always print on two different pages of paper.

The auto discovered printer is:

Driver: Brother MFC-L3770CDW series, driverless, cups-filters 1.28.15 (color, 2-sided printing)
Connection : implicitclass://Brother_MFC_L3770CDW_series/

I'm not able to delete that printer, it is always reappearing up on delete (Normal behaviour).

I installed a new one with cups web interface: http://localhost:631/printers/

Driver: MFC-L3770CDW series - IPP Everywhere (color, 2-sided printing)
Connexion : ipp://Brother%20MFC-L3770CDW%20series._ipp._tcp.local/

With this printer, I'm able to perform duplex printing.

**Expected behavior**
Duplex printing with driverless printers out of the box

**System Information:**
 - Mint Cinnamon 21.1
 - Xreader, LibreOffice
 - CUPS 2.4.1

Thanks for your implication in this project and making Linux always better;)

**Comment by lafontaj** [comment]
Hi,
Thanks for your detailed answer.
Unfortunately, I'm unable to reproduce the original problem, the printer is now printing correctly in duplex mode.   I don't know what to say, it looks like a user problem, but I did not change a thing on the configuration, but many Mint updates have been applied since my observation.

I did a lot of verifications (checking my back) to be sure I was using the right printer for my testing before posting, I did not want to overload your work with a non problem...apparently I just did it.  I'm very sorry about that.
